ext/reflection/php_reflection.stub.php

```diff
@@ -205,6 +205,8 @@ class ReflectionMethod extends ReflectionFunctionAbstract
 
     public function __construct(object|string $objectOrMethod, ?string $method = null) {}
```

Comment:
According to RFC the last argument should be required

Reply:
Is there follow-up for that? or passing one argument will remain til 8.4 as freezein action?
 https://wiki.php.net/rfc/deprecate_functions_with_overloaded_signatures#reflectionmethodconstruct

Reply:
The RFC lists the last parameter as required only in one of the signatures, but since there is another one with 1 parameter, the stub - which unified the two signatures - displays the 2nd parameter as optional.

> Is there follow-up for that?

What kind of follow-up do you mean? According to the RFC, the signatures and the behavior of ReflectionMethod::__construct() won't change until 9.0, except for the signature with 1 parameter will be deprecated in PHP 8.4. In 9.0, the constructor will only accept 2 arguments.

Reply:
Thank you! misread 

> deprecate the second constructor signature in PHP 8.4